### PR TITLE
Log E2E Test Result and URL to release tracker

### PR DESCRIPTION
**Pull request by hsubox76**
Send the E2E test result to the release tracker. Put the release tracker functions URL in a secret.

### CODE REVIEW

File: scripts/ci/notify-test-result.js [hunk 1]
```diff
@@ -50,34 +50,65 @@ async function notifyTestResults() {
...
+    req.write(
+      JSON.stringify({
+        text: message
+      })
+    );
+    req.end();
+  });
+
+  const logPromise = new Promise((resolve, reject) => {
+    const testStatus = status === 'succeeded' ? 'pass' : 'fail';
```
Comment (Feiyang1) [inline scripts/ci/notify-test-result.js:84]: where is `status` defined?

Comment (hsubox76) [reply scripts/ci/notify-test-result.js:84]: line 32

File: .github/workflows/e2e-test.yml [hunk 2]
```diff
@@ -19,57 +19,59 @@ jobs:
         working-directory: './e2e'
 
     steps:
-    - name: Checkout Repo
```
Comment (allspain) [inline .github/workflows/e2e-test.yml:22]: can you point out what actually changed? as A newcomer to this it's a little hard to follow

Comment (allspain) [reply .github/workflows/e2e-test.yml:22]: Actually it looks like it's just the release tracker url?

**Review by allspain — CHANGES_REQUESTED** [review]
If you can just answer the questions that would be helpful to finish the review

**Review by Feiyang1 — APPROVED** [review]

File: .github/workflows/e2e-test.yml (hunk 2 above)
Comment (hsubox76) [reply .github/workflows/e2e-test.yml:22]: Sorry, everything is in the diff because of applying auto formatting that changed all the indents. I think it is just adding the release tracker url. Most of the other changes are in the script itself.

**Review by allspain — APPROVED** [review]
